content/en/docs/faq.md

```diff
@@ -75,3 +75,34 @@ Not natively supported but a user can configure `revocationValidations` to `skip
 **A:** Ideally, we should validate the signing identity first and then use the public key in the signing identity to validate the artifact signature.
 However, this will lead to poor performance in the case where the signature is not valid as there are lots of validations against the signing identity including network calls for revocations, and possibly we won't even need to read the trust store/trust policy if the signature validation fails.
 Also, by validating artifact signature first we will still fail the validation if the signing identity is not trusted.
+
+## [Notary Project Terms](https://notaryproject.dev/docs/glossary/)
```

Comment:
Should we use a relative link for the glossary page?

Reply:
I tried the relative link but got an error during the process. I tried this here: `({{< relref "/docs/glossary" >}})`

Reply:
Maybe you can remove the relative link in the sub-heading and add a sentence in line 80, such as 
`The frequently asked questions about Notary Project terms are listed below. See the [glossary]({{< ref "/docs/glossary" >}}) page to learn the detailed definition of each Notary Project term.`